csvcubed/csvcubed/writers/urihelpers/skoscodelist.py

```diff
@@ -17,9 +18,24 @@ class SkosCodeListNewUriHelper:
     """
 
     code_list: NewQbCodeList
+    default_uri_style: URIStyle
 
     def _get_identifier_for_document(self) -> str:
-        return f"{self.code_list.metadata.uri_safe_identifier}.csv"
+        identifier = self.code_list.metadata.uri_safe_identifier
+
+        resolved_uri_style = self._resolve_uri_style()
+        if resolved_uri_style == URIStyle.Standard:
+            return identifier + ".csv"
+        elif resolved_uri_style == URIStyle.WithoutFileExtensions:
+            return identifier
+        else:
+            raise ValueError(f"Unhandled URI Style '{self.code_list.uri_style}'.")
+
+    def _resolve_uri_style(self) -> URIStyle:
+        if not (self.code_list is None or self.code_list.uri_style is None):
```

Comment:
I think I'd prefer `if self.code_list is not None and self.code_list.uri_style is not None` or the simplified `if self.code_list and self.code_list.uri_style` - I've been told (and believe) that negatives are harder to understand.

Reply:
fixed